docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v15.0.0 or later.
+- Docker version v(=docker.version=) or later.
+- A PostgresSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
```

Comment:
```suggestion
  - Access Graph needs a dedicated PostgresSQL compatible database to store its data.
```

should we refer as postgresql compatible database?

Reply:
I'm not so sure we should commit to "PostgreSQL-compatible", since there is no one definition of what that means. E.g. CockroachDB claims to be "compatible", but we have already found that TAG does not currently run on Cockroach.

Reply:
Btw the list sub-item talks about a "database" as an object inside of Postgres - one Postgres server/cluster can host many "databases", not a database as in an instance.

Reply:
I'd be careful as Cockarouch is Postgres compatible, but TAG won't work with it. I also don't know what other backends we will include with this phrase.

Reply:
I mean, not even cockroachdb say they are compatible with postgres 
> CockroachDB supports the [PostgreSQL wire protocol](https://www.postgresql.org/docs/current/protocol.html) and the majority of PostgreSQL syntax. This means that existing applications built on PostgreSQL can often be migrated to CockroachDB without changing application code.

They implement the wire protocol and some functionalities but they do not claim to be fully compatible